plugins/org.jboss.tools.ws.jaxrs.ui/src/org/jboss/tools/ws/jaxrs/ui/internal/importer/JaxRsConfigurator.java

```diff
@@ -63,8 +63,12 @@ public boolean canConfigure(IProject project, Set<IPath> ignoredDirectories, IPr
 			return false;
 		} finally {
 			try {
-				reader.close();
-				content.close();
+				if (reader != null) {
+				    reader.close();
```

Comment:
we fixed duplicated issue. 
However, here if `reader.close()` would fail, then `content` wouldn't be closed. See my PR for duplicated issue: https://github.com/jbosstools/jbosstools-webservices/pull/267/files